### PR TITLE
Reduce trace output

### PR DESCRIPTION
**Pull request by josegonzalez**
Cutting down on a few things that make `dokku trace` output annoying to read through.

### CODE REVIEW

**Comment by josegonzalez** [comment]
Of note: removing `TRACE=1` from the docker options during a trace run will shorten the output a further 3.5k lines.... Definitely worth making optional.

**Comment by josegonzalez** [comment]
Would be incredibly nice if we could refactor plugin commands such that they were native plugn commands as opposed to the case/switch setup we have now. I think that would decrease trace output tremendously. Potentially a bit late to do that, but thats up to you, @michaelshobbs.

Note: I'm happy to revert any of the commits in this PR.

**Comment by michaelshobbs** [comment]
Interesting. I'm down for reducing unnecessary trace output. What are you thinking in regards to using native plugn commands?

**Comment by josegonzalez** [comment]
Probably splitting out each command into a subcommand, like the old refactor PR that did exactly that. That would totally reduce the amount of output, though would be yet another refactor of plugins for 0.4.0 (and a much more backwards-incompatible change). Also not sure how we would use colons in filenames...

**Comment by michaelshobbs** [comment]
But how does that relate to "native plugn commands?"

**Comment by josegonzalez** [comment]
At the moment we send any commands through a `foreach plugin` loop, which adds a bunch of extra output we don't need. Instead we could just do `plugn trigger command_name` to trigger commands, right?

**Comment by michaelshobbs** [comment]
Yeah let me see what I can do. Not sure `trigger` is the right method since that goes through all enabled plugins.

**Comment by josegonzalez** [comment]
Note: that might not be the best route and we might still need to support both methods - the git plugin has `git-*`` as one of it's supported commands - though in the normal case should work quite well.

**Comment by michaelshobbs** [comment]
I say let's merge this and refactor plugin commands (again) afterwards

**Comment by josegonzalez** [comment]
Ha sorry :(

I'll rebase this then.

**Comment by josegonzalez** [comment]
Again, I don't know if refactoring how commands are triggered is the best way forward here. It would be the biggest bc-incompatible change we make thus far and potentially a lot of work. Would also basically break git history.

Then again, if it does look like it's worth it to reduce trace output, then we should break plugin compatibility in a single go. :(

**Comment by michaelshobbs** [comment]
This method is rough on maintenance to be sure because we'll need to maintain the list at the top of each file.

**Comment by michaelshobbs** [comment]
We're going to have some major conflicts between this PR and the nginx one....

**Comment by josegonzalez** [comment]
I'm happy to remove that check for now if that makes it easier to deal with.

**Comment by michaelshobbs** [comment]
It's fine. We can remove it when we refactor plugin commands. Both previous PRs we backwards compatible with the old method of command execution. So no need to hold `0.4.0` for it.

File: plugins/nginx-vhosts/functions [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  # Fixes using a wildcard dns service such as xip.io which allows for *.<ip address>.xip.io
+  echo "${RE_IPV4}\$"
+}
+
+get_ipv6_regex() {
+  local RE_IPV6="$(_ipv4_regex)"
+  # Ensure the ip address continues to the end of the line
+  # Fixes using a wildcard dns service such as xip.io which allows for *.<ip address>.xip.io
+  echo "${RE_IPV6}\$"
+}
```
Comment (michaelshobbs) [inline plugins/nginx-vhosts/functions]: I did something similar in #1476

**Comment by michaelshobbs** [comment]
@josegonzalez you want me to bring these commits into #1476 or just handle conflict after the fact?

**Comment by michaelshobbs** [comment]
closing in favor of #1476 which includes changes here

**Pull request closed without merging**